xbmc/windowing/GraphicContext.cpp

```diff
@@ -26,6 +26,8 @@
 
 #include <cassert>
 
+#include <cassert>
```

Comment:
no sure if it is just the github preview of the change, but the very same line seems to be already there two lines above ;)